<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="ogl_ir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="ogl_cam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_openIR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开红外相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_openCAM">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像机</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
